.travis/upload-nightly-wheels.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# This script gets newest wheels from S3 and uploads them PyPI
+
+WHEEL_DIR="./.whl"
+mkdir -p $WHEEL_DIR
+
+# Clone Ray to get commit hash
+git clone https://github.com/ray-project/ray.git
+
+# Sync newest wheels from AWS
+COUNT=0
+while ! [[ $WHEELS ]]; do
+    # Get commit hash
+    pushd ray
+    COMMIT_HASH=$(git rev-parse "HEAD~$COUNT")
+    popd
+
+    # Attempt to sync wheels
+    aws s3 sync --no-sign-request "s3://ray-wheels/$COMMIT_HASH/" $WHEEL_DIR
+    WHEELS=$(ls $WHEEL_DIR)
+
+    ((COUNT++))
+done
+
+# Rename wheels to ray_nightly-{version}.dev{date}-{info}.whl
```

Comment:
Will these be ordered properly based on upload date? That is, will `pip install ray-nightly` definitely get the most recent one?

Reply:
Not sure; we have to test this out (but seems like that's the case with tf_nightly).